### ISSUE TITLE
Dockerhub Readme

### ISSUE THREAD

**Issue by BenDaMAN303** [shawly/docker-ps3netsrv#2]
https://hub.docker.com/r/shawly/ps3netsrv

The README here says to use the environment variables "USER_ID" & "GROUP_ID"
Neither of those variable work. I have verified that my game folders/files are owned by my user, and that user id=1000, group=100.

I also tried PUID & PGID, which seemed to pass through to the container.
Here is what the log showed:

"[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 10-adduser: executing...
usermod: no changes

-------------------------------------
GID/UID
-------------------------------------

User uid: 1000
User gid: 100
-------------------------------------

[cont-init.d] 10-adduser: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done."

I have been able to get this working on Windows very easily, i also had it working under mccloud's release on my Unraid server before they had their git account stolen, and then all templates were removed etc.

Can i please get some guidance in how this should be configured with the latest release?

**Comment by shawly** [comment]
Ah sorry man, I've reduced my whole build process and changed my Dockerfile, haven't tested it properly since I'm not playing any PS3 lately, so this looks like a bug rather than misconfiguration.
I recommend using the previous version or change your UID/GIDs for the moment since it'll take some time until I find some free time to add this back in in the new version.

**Comment by shawly** [comment]
Finally got time, I'm currently reworking the docker build and I'm trying to fix the uid issue as well. Should be ready soon.

**Comment by shawly** [comment]
Alrighty, new release v1.3.0 is out, also contains a fix for the ps3netsrv itself from the main repo.
Setting ownership now works like the old way again with **USER_ID** and **GROUP_ID**, I changed the vars back to the old ones so migrating from v1.1.1 should be as easy as pulling the latest image.

This time I tested the image propely with my PS3, so it should be working properly now. Sorry for the inconvenience I caused with the rushed release from last time.
